mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -6,6 +6,9 @@ pipeline {
             label 'rocm'
         }
     }
+    environment {
+        PATH="$PATH:/opt/rocm/llvm/bin"
```

Comment:
I'm not sure this is required to build/check-mlir. I think this is required on `Static Test` stage. If that's the case, could you add a comment to be explicit on which step need this environment?

Reply:
I figured  that, since it's used all over the place, this should be placed on the top to avoid duplication (especially since it won't break anything)